### PR TITLE
Create diversity.md

### PR DESCRIPTION
**Pull request by tashimi**
First pass at a run book for this 

Signed-off-by: Tasha Drew <tasha.drew@gmail.com>

### CODE REVIEW

**Comment by parispittman** [comment]
appreciate this @tashimi!! thank you - amazing first pass!! 🚀 

/lgtm
/approve

**Comment by parispittman** [comment]
oh dang the spelling check didn't pass -

events/events-team/diversity.md:4:103: "contributers" is a misspelling of "contributors"
events/events-team/diversity.md:29:139: "Contributer" is a misspelling of "Contributor"
events/events-team/diversity.md:66:14: "refridgerator" is a misspelling of "refrigerator"

will happily approve after :)

**Comment by tashimi** [comment]
@parispittman hopefully gtg now! 

![tenor-168592661](https://user-images.githubusercontent.com/719646/60201987-8166dc00-97fe-11e9-9a9d-55a31db3e26c.gif

File: events/events-team/diversity.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+- Well organized, responsive via email, slack
+- Available to participate in weekly planning sessions 
+- Able to attend Contributor Summit 
+- Comfortable reaching out to community groups to encourage their members to attend and participate 
+
+## Activities  
+
+- Work with Leads cross-functionally to ensure accessibility, inclusiveness, and diversity are core to the event  
+- Use previous surveys, events, focus groups, and anything else needed to determine the success of previous events and how to improve for the next one 
+- Review the previous workplan and use feedback and datat to improve it for the upcoming event
```
Comment (ameukam) [inline events/events-team/diversity.md]:
```suggestion
- Review the previous workplan and use feedback and data to improve it for the upcoming event 
```

File: events/events-team/diversity.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+- Speakers and presenters should: 
+  * Take the CNCF Diversity training for speakers 
+  * Review the Critical Listening Guide https://www.ncwit.org/resources/critical-listening-guide 
+  * SIG updates should include a plan for how each SIG will be reaching out to and mentoring underrepresented people to increase inclusion in the projects 
+  * There should be no panels or groups of >3 speakers that are entirely overrepresented people
+- Summit location check list: 
+  * The website and physical conference site is welcoming to people with disabilities
+  * Make sure there is a microphone for all speakers
+  * Avoid putting panelists on a stool or on a high platform stage
+  * Ensure your check-in table allows attendees in wheelchairs to be greeted at eye-leve
```
Comment (ameukam) [inline events/events-team/diversity.md]: `eye-level` ?

File: events/events-team/diversity.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+- Use previous surveys, events, focus groups, and anything else needed to determine the success of previous events and how to improve for the next one 
+- Review the previous workplan and use feedback and datat to improve it for the upcoming event 
+- Create a work plan for the upcoming event, review it with the events team for feedback and consensus  
+- Execute the workplan, working cross-functionally with other leads and the CNCF Events team
+
+### Work Plan Items 
+
+- Pre-event: 
+  * Reach out to under-represented group organizations to encourage and invite people to attend both the new Contributor workshops and the Contributor Summit, as appropriate. Remember a personal invitation, making yourself available, and encouragement can go a long way, versus simply blasting a message out to an email distribution and calling it a day
+    - GoBridge
```
Comment (ameukam) [inline events/events-team/diversity.md]:
```suggestion
    - GoBridge: https://github.com/gobridge/about-us
```

**Comment by jdumars** [comment]
Thank you for this! 
/lgtm

**Pull request merged**